packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
 import * as React from 'react';
-import { IStyle, ITheme } from '../../Styling';
-import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+import { IStyle, ITheme } from '@uifabric/styling';
```

Comment:
Same comment here for `@uifabric/styling`.

Reply:
No this is right too